lib/Command/ExpirePassword.php

```diff
@@ -88,6 +90,11 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 			return 67;
 		}
 
+		if (!$user->canChangePassword()) {
+			$output->writeln("<error>Passwords cannot be expired for user: $uid</error>");
```

Comment:
should we add a reason: because the user is in a backend that doesn't support password change ?

Reply:
Why not.
I mentioned this sort of thing in https://github.com/owncloud/password_policy/issues/29#issue-339683422 (item 2 at the bottom of the post). In core, it would be nice to make some more friendly messages when password change is attempted for these type of users.